SU-HSDO/suhumsci
# SHS-4903: CKEditor5: Anchor Link plugin

# READY FOR REVIEW

## Summary
Installs enables CKEditor 5 anchor link plugin and enables it in every editor.

## Steps to Test
1. Check that `anchor_link` module is installed in `admin/modules`
2. Check that all editor toolbars in `admin/config/content/formats` have the `anchor_link` plugin enabled  (flag icon)
3. Create a new page at `node/add/hs_basic_page`
4. Add content and an anchor link
5. Save page
6. Check that adding the anchor to the new page url navigates to the correct section

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
